### PR TITLE
Fix indent for sequence

### PR DESCRIPTION
**Pull request by huacnlee**

### CODE REVIEW

File: tests/fixtures/bad_cases.expected.pest [hunk 1]
```diff
@@ -1,9 +1,9 @@
 params    = { "something" }
 arguments = {
-    "(" ~ (params // bad comment
-   ~ ("," ~ params)* // bad comment
-   // comment dont know if this was a problem.
-   ~ ","?)? ~ ")"
```
Comment (huacnlee) [inline tests/fixtures/bad_cases.expected.pest:6]: Before I fix, here have a extra space.

File: src/lib.rs [hunk 2]
```diff
@@ -37,14 +37,12 @@ pub struct Formatter<'a> {
...
     /// Indent space size
     indent: usize,
-    choice_first: bool,
-    sequence_space: usize,
 }
 
 impl<'a> Formatter<'a> {
     /// Create new formatter
     pub fn new(input: &'a str) -> Formatter<'a> {
-        Self { input, indent: 4, choice_first: true, sequence_space: 1 }
```
Comment (huacnlee) [inline src/lib.rs:47]: Remove this option key, we don't need it.

File: Cargo.toml [hunk 3]
```diff
@@ -4,6 +4,7 @@ description = "A formatter tool for pest"
 edition = "2018"
 license = "MPL-2.0"
 name = "pest_fmt"
+readme = "README.md"
 repository = "https://github.com/pest-parser/pest-fmt"
 version = "0.2.1"
```
Comment (tomtau) [inline Cargo.toml:9]:
```suggestion
version = "0.2.2"
```

**Review by tomtau — APPROVED** [review]
i guess some tests won't be checked on windows, but that's ok for now

**Comment by huacnlee** [comment]
I can find the error reason on Windows test, I have tried. 

That test is ok for Windows, not need to test. So we can pass for now.

**Pull request merged**